src/Pages/HomePage/HomePage.css

```diff
@@ -0,0 +1,79 @@
+.main-container {
+  padding: 2rem;
+}
+
+.original-img {
+  position: relative;
+}
+
+.small-img {
+  position: absolute;
+  top: 3.5rem;
+  left: 29rem;
+  width: 15.8rem;
+  height: 34rem;
+  animation-name: fade;
+  animation-timing-function: ease-in-out;
+  animation-iteration-count: infinite;
+  animation-duration: 4s;
+  animation-direction: alternate;
```

Comment:
You can use **animation** shorthand property here

Reply:
Done